### PR TITLE
Add bed-time-calculator extension

### PR DESCRIPTION
**Pull request by khlebobul**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/bed-time-calculator/package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    },
+    {
+      "name": "sleep-at",
+      "title": "Sleep At",
+      "description": "I know when I'll fall asleep and want to know when to wake up",
+      "mode": "view"
+    },
+    {
+      "name": "wake-up-at",
+      "title": "Wake Up At",
```
Comment (pernielsentikaer) [inline extensions/bed-time-calculator/package.json:27]:
```suggestion
      "title": "Wake up At",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Maybe you could make it even faste/better by using [arguments](https://developers.raycast.com/information/lifecycle/arguments) like this

Raycast 2024-06-11 at 15 07 41


I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/bed-time-calculator/package.json (hunk 1 above)
Comment (khlebobul) [reply extensions/bed-time-calculator/package.json:27]: Good afternoon, you are right, that would be better. I will definitely familiarise myself with how to do this and send it for checking again, thanks for the recommendations.

Comment (pernielsentikaer) [reply extensions/bed-time-calculator/package.json:27]: Awesome 🔥

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**